packages/auto-approve/src/interfaces.ts

```diff
@@ -55,6 +55,8 @@ export interface Versions {
   oldMinorVersion: string;
   newMajorVersion: string;
   newMinorVersion: string;
+  oldShaOrRevTag?: string;
```

Comment:
Is there a need to overload this class? Can we have a separate class for a SHA diff. The function that parses the diff could return a `Versions | ShaDiff` type.

Reply:
Sounds good!